### PR TITLE
Save Code produces identical results as shown in UI

### PR DESCRIPTION
**Pull request by an-dr-eas-k**
### Current status:

- When decompiling an assembly and using the "Save Code" function to produce csproj or sln projects, only some default settings are applied, differing from those in the options dialog.
- Additionally, when using the powershell scripts to decompile code, default settings are hard coded.
- The `DebugInfoProvider` is lost when using "Save Code" from ILSpy or in the powershell cmdlets. 

### Enhancement from this pull request
- Decompiling a complete assembly to a csproj or sln structure respects the settings from ILspy dialog, and preserves the `DebugInfoProvider`.
- The powershell cmdlets either use the ilspy.xml settings in AppData directory, or allow specifying a custom xml config file.

### Addressed Issues
#1674
#1328
#1426

### CODE REVIEW

**Comment by christophwille** [comment]
You are moving references around (package as well as project refs), deleting projects, and more. Eg adding Mono.Cecil to ICS.Decompiler is a complete no-go.

**Comment by siegfriedpammer** [comment]
We would prefer to discuss the actual problem this pull request tries to fix in an issue. Could you please open an issue for this? As @christophwille already noted, moving types around should not be done without prior careful analysis.

Thank you very much!

File: ICSharpCode.Decompiler.PowerShell/GetDecompilerCmdlet.cs [hunk 1]
```diff
@@ -15,20 +25,106 @@ public class GetDecompilerCmdlet : PSCmdlet
...
+
 		protected override void ProcessRecord()
 		{
 			string path = GetUnresolvedProviderPathFromPSPath(LiteralPath);
 
-			try {
-				var decompiler = new CSharpDecompiler(path, new DecompilerSettings() {
-					ThrowOnAssemblyResolveErrors = false
-				});
+			try
```
Comment (siegfriedpammer) [inline ICSharpCode.Decompiler.PowerShell/GetDecompilerCmdlet.cs]: This does not respect our code formatting style. Please make sure you follow the settings in .editorconfig.

**Comment by siegfriedpammer** [comment]
As discussed in #1674, the changes proposed in this pull request unfortunately cannot be merged.

**Pull request closed without merging**